agent/src/beerocks/slave/son_slave_thread.h

```diff
@@ -143,8 +142,8 @@ class slave_thread : public beerocks::socket_thread {
     void platform_manager_stop();
     void hostap_services_off();
     bool hostap_services_on();
-    bool ap_manager_start();
-    void ap_manager_stop();
+    bool ap_manager_start() { return true; };
```

Comment:
It looks a bit weird that these are nops.

Reply:
This is temporary so everything will be compiled successfully. It is removed few on a commit ahead.